### PR TITLE
[JBIDE-20463] reset token value when new connection is selected

### PR DESCRIPTION
**Pull request by fbricon**
Signed-off-by: Fred Bricon <fbricon@gmail.com>

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/ConnectionEditor.java [hunk 1]
```diff
@@ -223,8 +222,7 @@ private void disposeBindings() {
 	@Override
 	protected void onSelectedConnectionChanged(IObservableValue selectedConnection) {
 		IConnection conn = (IConnection) selectedConnection.getValue();
-		if(!(conn instanceof Connection)) return;
-		detailViewModel.setSelectedConnection((Connection)conn);
+		detailViewModel.setSelectedConnection(conn);
```
Comment (jcantrill) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/ConnectionEditor.java]: Is there a reason to remove the instance check?

**Comment by jcantrill** [comment]
@fbricon assuming it resolves the issue, it LGTM

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/ConnectionEditor.java (hunk 1 above)
Comment (fbricon) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/ConnectionEditor.java]: yes, setSelectedConnection takes an IConnection argument, and the instance check prevents NewConnectionMarker instances to be set, which leads to the UI not being reset.

**Comment by fbricon** [comment]
Applied in master

**Pull request closed without merging**